package.json

```diff
@@ -28,6 +28,7 @@
   },
   "dependencies": {
     "@ethersproject/abstract-provider": "^5.7.0",
+    "@hasankhadra/zero-wallet-server": "^1.0.3",
```

Comment:
probably best to rename the package to "@questbook/zero-wallet-server"

Reply:
I wasn't able to publish it with @questbook, I'll check with Madhavan what's needed to get this done